rdkit/Chem/Fingerprints/ClusterMols.py

```diff
@@ -18,16 +17,21 @@
     --actTable="dop_test" --actName="moa_quant"
 
 """
-from rdkit.Dbase.DbConnection import DbConnect
-from rdkit.Dbase import DbInfo, DbUtils
-from rdkit.ML.Data import DataUtils
-from rdkit.ML.Cluster import Clusters
+from __future__ import print_function
+
+import numpy
+
+from rdkit import DataStructs
+from rdkit.Chem.Fingerprints import FingerprintMols, MolSimilarity
 from rdkit.ML.Cluster import Murtagh
-import sys
 from rdkit.six.moves import cPickle
-from rdkit.Chem.Fingerprints import FingerprintMols, MolSimilarity
-from rdkit import DataStructs
-import numpy
+
+# from rdkit.Dbase.DbConnection import DbConnect
+# from rdkit.Dbase import DbInfo, DbUtils
+# from rdkit.ML.Data import DataUtils
+# from rdkit.ML.Cluster import Clusters
+# import sys
+
 _cvsVersion = "$Id$"
```

Comment:
Wow, this is a horrible ancient source control dropping. 
I think we can get rid of everything between here and the definition of `message`. The usage of __VERSION_STRING way below in the "main" block can also be removed.